server/form-pages/apply/further-considerations/roomSharing.test.ts

```diff
@@ -24,7 +24,7 @@ describe('RoomSharing', () => {
     })
   })
 
-  itShouldHaveNextValue(new RoomSharing({}), '')
+  itShouldHaveNextValue(new RoomSharing({}), 'vulnerability')
```

Comment:
I think this has snuck in from another commit

Reply:
Good catch!